src/UI/COStateInfo.java

```diff
@@ -114,10 +122,10 @@ public String getFullStatusLabels()
     sb.append("Unit count:")   .append("\n");
     sb.append("Vehicle count:").append("\n");
     sb.append("Unit funds:")   .append("\n");
-    if( showAbilityInfo )
+    for( Commander co : currentEnergy.keySet() )
     {
-      sb.append("Ability Power:").append("\n");
-      sb.append("Next Ability:") .append("\n");
+      sb.append(co.coInfo.name + " Energy:").append("\n");
+      sb.append(" - Next Energy:") .append("\n");
```

Comment:
Why the change from "Ability" to "Energy"? I think the former sounds more natural.